flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -203,4 +203,28 @@ public class KubernetesOperatorConfigOptions {
                     .defaultValue(false)
                     .withDescription(
                             "Enables dynamic change of watched/monitored namespaces. Defaults to false");
+
+    public static final ConfigOption<Duration> OPERATOR_RETRY_INITIAL_INTERVAL =
+            ConfigOptions.key("kubernetes.operator.retry.initial.interval")
+                    .durationType()
+                    .defaultValue(Duration.ofSeconds(5))
+                    .withDescription(
+                            "Initial interval of automatic reconcile retries on recoverable errors. Defaults to "
+                                    + Duration.ofSeconds(5));
+
+    public static final ConfigOption<Double> OPERATOR_RETRY_INTERVAL_MULTIPLIER =
+            ConfigOptions.key("kubernetes.operator.retry.interval.multiplier")
+                    .doubleType()
+                    .defaultValue(2.0)
+                    .withDescription(
+                            "Interval multiplier of automatic reconcile retries on recoverable errors. Defaults to "
+                                    + 2.0);
+
+    public static final ConfigOption<Integer> OPERATOR_RETRY_MAX_ATTEMPTS =
+            ConfigOptions.key("kubernetes.operator.retry.max.attempts")
+                    .intType()
+                    .defaultValue(10)
+                    .withDescription(
+                            "Max attempts of automatic reconcile retries on recoverable errors. Defaults to "
+                                    + 10);
```

Comment:
You don't need to specify default values in the description itself. It is generated automatically in the HTML